cf/api/fakes/fake_service_key_repo.go

```diff
@@ -1,25 +1,44 @@
 package fakes
 
+import (
+	"github.com/cloudfoundry/cli/cf/models"
+)
+
 type FakeServiceKeyRepo struct {
-	CreateServiceKeyError error
+	CreateServiceKeyMethod CreateServiceKeyType
+	ListServiceKeysMethod  ListServiceKeysType
+}
 
-	CreateServiceKeyArgs CreateServiceKeyArgsType
+type CreateServiceKeyType struct {
+	InstanceId string
+	KeyName    string
+
+	Error error
 }
 
-type CreateServiceKeyArgsType struct {
-	ServiceInstanceId string
-	ServiceKeyName    string
+type ListServiceKeysType struct {
+	InstanceId string
+
+	ServiceKeys []models.ServiceKey
+	Error       error
 }
 
 func NewFakeServiceKeyRepo() *FakeServiceKeyRepo {
 	return &FakeServiceKeyRepo{
-		CreateServiceKeyArgs: CreateServiceKeyArgsType{},
+		CreateServiceKeyMethod: CreateServiceKeyType{},
+		ListServiceKeysMethod:  ListServiceKeysType{},
 	}
 }
 
 func (f *FakeServiceKeyRepo) CreateServiceKey(instanceId string, serviceKeyName string) (apiErr error) {
```

Comment:
In general, we shy away from naming return variables, as this can lead to confusing code. Also, in this specific case, apiErr is not even being used at all, so it's better to just leave it as an unnamed error.